### ISSUE TITLE
Host CPU usage being capped by WSL2 hypervisor

### ISSUE THREAD

**Issue by rmsilva1973** [microsoft/WSL#5174]
* Your Windows build number:  **19619.1000**

* What you're doing and what's happening:  I'm building a few projects in parallel (namely the  Linux Kernel and another opensource projects in C++) on Ubuntu 20.04 runing under WSL2 with Linux kernel **4.19.84-microsoft-standard+** and even tough the linux vm shows 100% cpu usage, the host CPUs won't go anywhere over 50% (see screenshot below)

* What's wrong / what should be happening instead:
It seems like WSL2 is capping CPU usage for the Ubuntu VM. There should be a way to let it use all my CPU available.

![image](https://user-images.githubusercontent.com/28487379/81022099-23a21800-8e43-11ea-8ad2-b90a3a0a5669.png)

**Comment by bpulliam** [comment]
Adjust .wslconfig file in increase processors, in your case 4 (# of logical processors). Marking as duplicate of #6033
